guides/user/canary/judge/index.md

```diff
@@ -0,0 +1,80 @@
+both deployments are compared in order to check for significant degradation.
+This is done in two phases:
+
+* metric collection (retrieval)
+
+  This phase retrieves the key metrics from the baseline and canary deployments.
+  These metrics are typically stored in a time-series database, and include a
+  set of tags or annotations that identify which deployment the data was
+  collected from (canary or baseline).
+
+* judgment
+
+  In this phase Spinnaker compares those metrics, rendering a decision to pass
+  or fail the canary (that is, was there was a significant degradation in the
+  metrics?) This judgment consists of four main steps, and they're described
+  below.
+
+
+## Step 1: Data validation
+
+Data validation ensures that there's data for the baseline and canary metrics
+before analysis begins.
+
+If metric collection returns an empty array for either the baseline or canary
+metric (or both) the metric is labeled `NODATA` and analysis moves on to the
+next metric.
+
+Note that some metrics might have no data, but for good reasons. For example, an
+error counter has no value if there are no failures. For this reason the judge
+does not automatically fail the canary for `NODATA`.
+
+## Step 2: Data cleaning
+
+This step prepares the raw metrics for comparison. This entails handling missing
+values from the input.
+
+Note that there are different strategies for handling missing values based on
+the type of metric. For example, missing values, represented as NaNs, may be
+replaced with zeros for error metrics while they may be imputed or removed for
+other types of metrics. In addition, the data cleaning step can, optionally,
+remove outliers from the data before comparison.
+
+## Step 3: Metric comparison (classification)
+
+This is the step that compares the canary and baseline data for each included
+metric. The output of this step is a classification, for each metric indicating
+if there is a significant difference between the canary and baseline.
+
+Each metric is classified as either “Pass”, “High”, or “Low” as shown in the
+screen shot below.
+
+<!--- insert image here, if available --->
+
+The primary metric comparison algorithm (classifier) in Kayenta uses a
+nonparametric statistical test to check for a significant difference between the
+canary and baseline metrics.
+
+## Step 4: Score computation
+
+After each metric is classified, a final score is computed which represents how
+similar the canary is to the baseline. This score is calculated as the ratio of
+"Pass" metrics out of the total number of metrics.
+
+For example, if 9 of 10 metrics are classified as “Pass” the score is 90%. The
+threshold score for overall canary pass, marginal, or fail is
+
+While there are more complex scoring methodologies, we bias toward techniques
+that are simple to interpret and understand; understanding why a decision was
+made is important to the success of Automated Canary Analysis.
```

Comment:
This wording feels like it belongs in a blog post rather than the technical docs

Reply:
Yes, I see what you're saying. 

Because I'm going to close this pr and create another, I'm going to leave this in for now. I'll try to remember to @ Chris on this to see how he feels about it. It's his wording.

Reply:
Since the system supports pluggable judges, and this is the out-of-the-box judge, couldn't hurt to describe the motivation behind its design. Maybe it can be reworded as a summary of this judge?

Reply:
@matt Are we referring to this judge by name? "Kayenta judge"? "Default judge"?

Reply:
It is the `NetflixACAJudge`.